server/manager/src/main/java/org/apache/accumulo/manager/upgrade/Upgrader12to13.java

```diff
@@ -114,6 +118,8 @@ public void upgradeMetadata(ServerContext context) {
     removeCompactColumnsFromTable(context, AccumuloTable.METADATA.tableName());
     LOG.info("Removing bulk file columns from metadata table");
     removeBulkFileColumnsFromTable(context, AccumuloTable.METADATA.tableName());
+    LOG.info("Removing problems reports from metadata table");
+    removeMetadataProblemReports(context);
```

Comment:
Should we show the existing problems if any are found, or should we require that no problems exist before an upgrade? (I'm leaning towards the former).

My concern would be that an upgrade erases evidence of an unresolved problem that should be addressed.

Reply:
We could log problem reports that are removed

Reply:
It would be tricky to require no problem reports because some are in the metadata table and some are in zookeeper.  To read the metadata table we need to upgrade zookeeper, so problem reports in the metadata table would only be seen after upgrading ZK.  Would not want to fail after partially upgrading when detecting problem reports.  So will look in to logging them.  May need to bring back the code to deserialize them, could put that in upgrade code.

Reply:
Updated to log the problem reports being deleted in 06499dd7a0.  Copied the minimal code needed from the old ProblemReport class to do the logging.

Reply:
Logging the ones found and removed is totally fine.